prometheus-to-sd/README.md

```diff
@@ -52,6 +52,29 @@ E.g, if `monitored-resource-type-prefix` = "k8s_", metrics will be written to "k
 If `namespace-id` and `pod-id` are specified, metrics will be written to "k8s_pod" type, with additional resource labels `namespace_name` and `pod_name`.
 If Prometheus-to-sd is configured to scrape container name from metrics, metrics will be written to "k8s_container" type, with extra resource label "container_name".
 
+From 0.9.0, custom resource type and labels can be specified per source config.
+Within the `source` URL, two new flags are supported:
+- `customResourceType` specifies the resource type override. When this flag is
+  present, it will override other resource type settings (e.g.
+  `monitored-resource-type-prefix`)
+- `customLabel[<label_name>]` specifies the labels that are provided for given
+  `customResourceType`. If `customResourceType` is not specified, this field is
+  ignored. All the labels for custom resource type has to be explicitly listed
+  in the source config. Omitting the value for the following labels will cause
+  prometheus-to-sd to use a value from GCE config:
+  - `instance_id`
+  - `cluster_name`
+  - `project_id`
+  - `location`
```

Comment:
Maybe it would be worth adding 'node_name' as well which is used by k8s_node right now.

Reply:
Good point, done.